### DISCUSSION TITLE
Help us drive future product direction by joining Coder's alpha program

### DISCUSSION THREAD

**Discussion by nathanpotter** [coder/code-server#2137] [General]
We're working on some new products at Coder and we're looking for code-server community members who are interested in joining our alpha program to gain pre-release access and to help provide product feedback.

If you're interested in helping us drive the future of remote development, please fill out the form at the link below.
[Join our Alpha Program](https://codercom.typeform.com/to/U4IKyv0W)

**Edit:** This is now open for everyone to use. Please leave feedback on [our Slack](https://community.coder.com)

Try it out: https://github.com/cdr/code-server#alpha-program-

**Comment by sr229** [comment]
Awesome! Count me in. I was actually interested on the platform itself as well since I joined the public SaaS alpha and my time at Coder as well.

**Reply by schmidDan** [reply]
Sounds great! I took the tour through the alpha form as well 😉

**Comment by hackerpain** [comment]
Been using it. Its pretty awesome but faced an issue one time. Besides, its apparently spawning multiple instances of code-server -- I had one instance running on :8080 

Not sure what caused that. But only one time I faced an issue which said "everything" properly configured but can't connect to instance.

Cheers

**Comment by codercatdev** [comment]
I am in using it, not sure if I need accepted still :D

**Reply by bpmct** [reply]
nope, it is open to everyone now :)

**Reply by bpmct** [reply]
https://github.com/cdr/code-server#alpha-program-

**Comment by demyxco** [comment]
Interested!

**Comment by aralroca** [comment]
I just get this error message:

![image](https://user-images.githubusercontent.com/13313058/101211457-81b40980-3677-11eb-9fd2-9ed3851f66d3.png)

**Reply by code-asher** [reply]
Could you post some logs? Thank you!

For reference: 
https://github.com/cdr/code-server/blob/v3.7.4/doc/FAQ.md#how-do-i-debug-issues-with-code-server
